engine/src/main/java/org/destinationsol/common/SolDescriptiveException.java

```diff
@@ -5,22 +5,21 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.destinationsol.common;
 
-import java.lang.annotation.Documented;
-
-/**
- * Annotated variable can be null, param can be null, method can return null.
- */
-@Documented
-public @interface Nullable {
+public class SolDescriptiveException extends RuntimeException {
+    public SolDescriptiveException(String msg){
```

Comment:
`msg) {`
Notice the space.

Reply:
weird, IDEA usually puts in the space by itself. Quick Ctrl+Alt+L will do